app/lib/actions/candlepin/environment/set_content.rb

```diff
@@ -32,15 +36,24 @@ def finalize # rubocop:disable Metrics/AbcSize
               # Since its a dup id refresh the existing ids list (which hopefully will not have the duplicate content)
               # and try again.
               output[:add_ids] = content_ids - existing_ids
+            rescue RestClient::ResourceNotFound => e
+              # Set a higher limit for retries just in case the missing content is not being parsed from the error body correctly.
+              # If the content is not found after the retries, assume it is gone and continue.
+              raise e if ((retries += 1) == 1_000)
```

Comment:
Should this be `1000`?

Reply:
I like using the underscore to help with readability: https://docs.ruby-lang.org/en/3.2/syntax/literals_rdoc.html#label-Integer+Literals
It gets ignored by the interpreter.